### PR TITLE
Localization

### LINKED ISSUE #26: Setup i18next and localization.
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
We should be able to switch between 2 languages.

**Screenshots of the design/if any**
N/A

**Additional context**
Check out i18next.

### PR DESCRIPTION
**Pull request by ZhraaBaharAluloom**
Installed i18next, configured it and added Arabic and English translation to the available components (auth pages and footer)

### CODE REVIEW

File: src/components/Languages.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const Languages = () => {
+  return (
+    <select
+      className="my-5 outline-none rounded-sm lg:mt-16 self-center"
+      id="lang-dropdown"
+      onChange={(e) => i18next.changeLanguage(e.target.value)}
+    >
+      <option value="ar">🇸🇦 العربية</option>
+      <option value="en">🇬🇧 English</option>
+      <option value="kurd">🇮🇶 Kurdish</option>
```
Comment (RawandDev) [inline src/components/Languages.jsx]: It would be better if you add `defaultValue` to one of them when the page renders.

**Review by RawandDev — CHANGES_REQUESTED** [review]
Nice job Zahra. I noticed that after refreshing the page there is an issue. For example I choose English language and refresh the page. The dropdown is still Arabic but the language is English so it would be better if the dropdown changes even after refreshing.
I can also contribute to this PR if it is fine and @MuhannedNoman has no issue with it.

File: src/components/Languages.jsx (hunk 1 above)
Comment (ZhraaBaharAluloom) [reply src/components/Languages.jsx]: Sure

Comment (RawandDev) [reply src/components/Languages.jsx]: I also forgot to mention that you can use PR template like this:
## What?
A description of what you did in this PR
## Why?
A description of why you did this PR?
## How to use?
How can we use what you built here? If applicable. 
## Screenshots 
A clear screenshots of your work.
## Gif
A gif showing your work, if you want.

**Review by MuhannedNoman — DISMISSED** [review]
LGTM. Feel free to merge.

**Review by RawandDev — APPROVED** [review]
LGTM

**Pull request merged**